packages/mdc-ripple/_mixins.scss

```diff
@@ -224,12 +302,22 @@
   }
 }
 
-@mixin mdc-states-interactions_($color, $has-nested-focusable-element, $opacity-modifier: 0) {
-  @include mdc-states-base-color($color);
-  @include mdc-states-hover-opacity(mdc-states-opacity($color, hover) + $opacity-modifier);
+@mixin mdc-states-interactions_($color, $has-nested-focusable-element, $opacity-modifier: 0, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-feature-targets($feat-color) {
+    @include mdc-states-base-color($color);
+  }
+
+  @include mdc-feature-targets($feat-structure) {
```

Comment:
should this be color?

Reply:
I suspect it's structure because the emitted style isn't color related, but the logic is color-dependent... @mmalerba, thoughts on this? (This would apply to press/focus as well)

Reply:
Discussed offline and yes, this should be changed because it's color-dependent in typical use. Pushed a commit to fix it.